### PR TITLE
Migrate CI to Github actions

### PR DESCRIPTION
**Pull request by browniebroke**
As mentioned [here](https://github.com/django-compressor/django-compressor/pull/1003#issuecomment-756967919).

Due to Github actions limitation, new workflows added from a fork aren't running on pull requests, but one can see the result on the GH actions of my fork: https://github.com/browniebroke/django-compressor/actions/runs/738417695

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,42 +0,0 @@
...
-      env: TOXENV=py36-3.0.X
-    - python: 3.7
-      env: TOXENV=py37-3.0.X
-      dist: xenial
-    - python: 3.8
-      env: TOXENV=py38-3.0.X
-      dist: xenial
-
-notifications:
-  irc: "irc.freenode.org#django-compressor"
```
Comment (browniebroke) [inline .travis.yml:39]: Not sure how to replicate that, but there is an action which might help: https://github.com/marketplace/actions/notify-irc

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+      - ci/gh-actions
```
Comment (browniebroke) [inline .github/workflows/ci.yml:8]: This line should probably be removed before merging.


```suggestion
```

File: .travis.yml (hunk 1 above)
Comment (diox) [reply .travis.yml:39]: This can be dropped

**Comment by carltongibson** [comment]
Hi @browniebroke thanks for this. Let me put it on the list for the week to look at. 👍

**Review by diox — APPROVED** [review]

**Pull request merged**